ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -513,40 +513,40 @@ object Config {
           )
 
         // TODO append-only: remove after removing support for the current (mutating) schema
```

Comment:
Do you still need to keep this TODO around?

Reply:
mutating schema is still around. We will be able to get rid of i when everyone starts using append-only